packages/contracts/contracts/plugin/PluginSetupProcessor.sol

```diff
@@ -123,19 +150,13 @@ contract PluginSetupProcessor is DaoAuthorizable {
     /// @notice Thrown if a plugin repository does not exist on the plugin repo registry.
     error PluginRepoNonexistent();
 
-    /// @notice Thrown if a plugin setup is not prepared.
-    /// @param setupId The abi encoded hash of versionTag & permissions & helpers.
-    error SetupNotPrepared(bytes32 setupId);
-
     /// @notice Thrown if a plugin setup was already prepared.
-    /// @param setupId The abi encoded hash of versionTag & permissions & helpers.
+    /// @param setupId Already prepared setupId.
     error SetupAlreadyPrepared(bytes32 setupId);
 
-    /// @notice Thrown if a plugin setup is not applied.
-    error SetupNotApplied();
-
-    /// @notice Thrown if a plugin setup was already prepared. This is done in case the `PluginSetup` contract is malicios and always/sometime returns the same addresss.
-    error SetupAlreadyApplied();
+    /// @notice Thrown when setup is no longer eligible for the `apply`. This could happen if another prepared setup was chosen for the apply or setupId wasn't prepared in the first place.
```

Comment:
```suggestion
    /// @notice Thrown when the setupId is no longer eligible for the `apply`. This could happen if another prepared setupId was chosen for the apply or setupId wasn't prepared in the first place.
```